ui/src/institutions/components/InstitutionAddress.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Map, List } from 'immutable';
+import InlineList, {
+  InlineUL,
+  SEPARATOR_COMMA,
+} from '../../common/components/InlineList';
+
+function postalAddressContainsString(postalAddress, string) {
+  const postalAddressString = postalAddress.join(' ');
+  return postalAddressString.toLowerCase().indexOf(string.toLowerCase()) > -1;
+}
+
+function InstitutionAddress({ address }) {
+  const postalAddress = address.get('postal_address', List());
+  const city = address.getIn(['cities', 0]);
+  const country = address.get('country');
+  return (
+    <span>
+      <InlineUL separator={SEPARATOR_COMMA} wrapperClassName="di">
+        {postalAddress.size > 0 && <InlineList items={postalAddress} />}
```

Comment:
this check should be already in `InlineList`

Reply:
It is already in ```InlineList``` but if I don't add this check also here then it doesn't seem to work with the ```InlineUL```. If I have an empty list and then something after it, it will still add the separator in the middle unless I add the condition here as well